doc/demo.md

```diff
@@ -66,6 +66,8 @@ new tick, it looks like follows (sizes specified in bits):
 `is_tick` is set to `0` for normal chunks.
 
 `type` can be one of `1` (snapshot), `2` (message) or `3` (snapshot delta).
+The `chunk_data` of snapshots and snapshot deltas goes through variable-int
+compression once, and twice for messages.
```

Comment:
Does "twice" mean decoded as ints twice, and the resulting bytes are then unpacked using TW variable int encoding a third time if the message has an int?

Or does it only go through int decoding once if the message doesn't contain any integers?